sources/Interop/Windows/shared/uuids/Windows.Manual.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from shared/uuids.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics.Contracts;
+using System.Runtime.CompilerServices;
+
+namespace TerraFX.Interop
+{
+    public static partial class Windows
+    {
+        /// <summary>
+        /// Retrieves the GUID of of a specified type.
+        /// </summary>
+        /// <typeparam name="T">The type to retrieve the GUID for.</typeparam>
+        /// <returns>
+        /// A <see cref="GUID"/> value wrapping a pointer to the GUID data for the
+        /// input type. This value can be either converted to a <see cref="Guid"/>
+        /// pointer, or implicitly assigned to a <see cref="Guid"/> value.
+        /// </returns>
```

Comment:
The rest of the repo, where applicable, uses single line summaries and doesn't do line wrapping.
```suggestion
        /// <summary>Retrieves the GUID of of a specified type.</summary>
        /// <typeparam name="T">The type to retrieve the GUID for.</typeparam>
        /// <returns>A <see cref="GUID"/> value wrapping a pointer to the GUID data for the input type. This value can be either converted to a <see cref="Guid"/> pointer, or implicitly assigned to a <see cref="Guid"/> value.</returns>
```